github input panics when repo has no language

### Relevant telegraf.conf:
<!-- [Place config betweetn triple backticks '```'] -->
```toml
[[inputs.github]]
  ## List of repositories to monitor
  repositories = [
        "influxdata/community"
  ]

  ## Github API access token.  Unauthenticated requests are limited to 60 per hour.
  access_token = "<SOMETHING>"

  ## Timeout for HTTP requests.
  # http_timeout = "5s"
```

### System info:

master branch, mac os

### Steps to reproduce:

1. Start Telegraf with this config, telegraf panics

### Expected behavior:
Telegraf should not panic

### Actual behavior:
Telegraf panics

### Additional info:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x4b6ddca]

goroutine 5820 [running]:
github.com/influxdata/telegraf/plugins/inputs/github.getTags(...)
	/Users/rsavage/go/src/github.com/influxdata/telegraf/plugins/inputs/github/github.go:163
github.com/influxdata/telegraf/plugins/inputs/github.(*GitHub).Gather.func1(0xc0013a7aa0, 0xc000160380, 0x669cee0, 0xc0000dc008, 0xc00015a561, 0x14, 0x66b8420, 0xc00019c100)
	/Users/rsavage/go/src/github.com/influxdata/telegraf/plugins/inputs/github/github.go:130 +0x32a
created by github.com/influxdata/telegraf/plugins/inputs/github.(*GitHub).Gather
	/Users/rsavage/go/src/github.com/influxdata/telegraf/plugins/inputs/github/github.go:106 +0x136
```

--- a/plugins/inputs/github/github.go
+++ b/plugins/inputs/github/github.go
@@ -148,9 +148,9 @@ func splitRepositoryName(repositoryName string) (string, string, error) {
 	return splits[0], splits[1], nil
 }
 
-func getLicense(repositoryInfo *github.Repository) string {
-	if repositoryInfo.GetLicense() != nil {
-		return *repositoryInfo.License.Name
+func getLicense(rI *github.Repository) string {
+	if licenseName := rI.GetLicense().GetName(); licenseName != "" {
+		return licenseName
 	}
 
 	return "None"
@@ -158,19 +158,19 @@ func getLicense(repositoryInfo *github.Repository) string {
 
 func getTags(repositoryInfo *github.Repository) map[string]string {
 	return map[string]string{
-		"owner":    *repositoryInfo.Owner.Login,
-		"name":     *repositoryInfo.Name,
-		"language": *repositoryInfo.Language,
+		"owner":    repositoryInfo.GetOwner().GetLogin(),
+		"name":     repositoryInfo.GetName(),
+		"language": repositoryInfo.GetLanguage(),
 		"license":  getLicense(repositoryInfo),
 	}
 }
 
 func getFields(repositoryInfo *github.Repository) map[string]interface{} {
 	return map[string]interface{}{
-		"stars":       *repositoryInfo.StargazersCount,
-		"forks":       *repositoryInfo.ForksCount,
-		"open_issues": *repositoryInfo.OpenIssuesCount,
-		"size":        *repositoryInfo.Size,
+		"stars":       repositoryInfo.GetStargazersCount(),
+		"forks":       repositoryInfo.GetForksCount(),
+		"open_issues": repositoryInfo.GetOpenIssuesCount(),
+		"size":        repositoryInfo.GetSize(),
 	}
 }
 

